docs/application/dotnet/guides/nui/nui-components/Pagination.md

```diff
@@ -1,161 +1,262 @@
 # Pagination
 
-Pagination is a common component supporting multiple pages. It shows the number of pages available and the currently active page.
+
+Pagination is a component for making navigation through multipage windows more user-friendly.
```

Comment:
```suggestion 
Pagination is a component for making navigation through multipage windows more user-friendly. It can be used to display the number of available pages and marking the currently viewed one. The following figure illustrates the use of simple pagination. It shows 5 available pages, of which the first one is currently displayed.